contribute/e2e/CONTRIBUTING.md

```diff
@@ -0,0 +1,113 @@
+## INTRODUCTION TO OPENEBS E2E
+
+OpenEBS e2e is focused on workload simulation & application deployment on OpenEBS storage, predominantly in a 
+kubernetes environment. In its current state, it includes application workflow tests,basic compliance and resiliency tests. 
+The scope of the said tests is expected to evolve over time to include performance and security tests as well. 
+The e2e is mostly written in ansible, i.e., as ansible playbooks with docker used for test images (these include standard 
+application images from dockerhub as well as custom-built images stored in openebs/test-storage).Since the tests are performed 
+in a kubernetes environment, the test images are typically deployed as "pods" and "jobs" and therefore accompanied by the 
+respective specification YAML files.
+
+## BUILDING BLOCKS OF OPENEBS CI
+
+The OpenEBS Continuous Integration has two distinct parts : 
+
+- Travis is used to run unit tests, build code and push images to dockerhub (in case of repos where the image is the build artifact)

```

Comment:
repos = repositories??
the image = image

Reply:
Done